cmd/run/run.go

```diff
@@ -685,11 +687,15 @@ func (s *ServerContext) Run(ctx context.Context, config *Config) error {
 
 	unaryInterceptors := []grpc.UnaryServerInterceptor{
 		requestid.NewUnaryInterceptor(),
+		grpc_validator.UnaryServerInterceptor(),
+		validator.UnaryServerInterceptor(),
```

Comment:
I debated on just copy/paste the `grpc_validator` code into the `validator` package and combining these middlewares into 1, because of the dependency on order between the two of them. But I ended up choosing this route to avoid code duplication.

If ya'll feel differently let me know and I'll change it.

Reply:
I think i prefer having `grpc_validator.UnaryServerInterceptor(),` embedded so that we hide that complexity from here. 

We can also leave as is and put a comment that `validator` must always come after `grpc_validator`. But this is more flaky

Reply:
I'm not sure how we can embed it, but we can copy the behavior into the `validator` package instead.

Reply:
Refactored to embed it. See https://github.com/openfga/openfga/pull/998/commits/fb765da5eeaeef2673fc8fa09ab14cefcddbfb6e